.github/workflows/ci.yml

```diff
@@ -85,6 +85,10 @@ jobs:
       - uses: docker/setup-qemu-action@v2
         with:
           platforms: arm64,ppc64le
+      - name: Set up Docker Buildx
```

Comment:
I am not well-versed in buildx.
if you want me to continue the review of this PR, I would recommend splitting this out to a follow-up